geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/ingest/ScaldingDelimitedIngestJob.scala

```diff
@@ -160,42 +160,75 @@ class ScaldingDelimitedIngestJob(args: Args) extends Job(args) with Logging {
   // Populate the fields of a SimpleFeature with a line of CSV
   def ingestDataToFeature(line: String, feature: SimpleFeature) = {
     val reader = CSVParser.parse(line, delim)
-    val fields: List[String] =
+    val csvFields =
       try {
-        val allFields = reader.getRecords.get(0).toList
-        if (colList.isDefined) colList.map(_.map(allFields(_))).get else allFields
+        reader.getRecords.get(0).toList
```

Comment:
we actually have never supported a header with ingest tools...but the next ticket michael matthews is working on fixes that.